Resources/views/full/place.html.twig

```diff
@@ -7,17 +7,23 @@
         </div>
     </div>
     <div class="row">
-        {% if not ez_is_field_empty( content, 'image' ) %}
+        {% if ( relatedImage is empty ) and (ez_is_field_empty( content, 'image' )) %}
```

Comment:
for consistency, could we have spaces before closing and after opening parenthesis all the way?